docs/playbook/ingesting-data/005-ingesting-dynamo-db-tables.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Ingesting Dynamo DB tables into the Landing Zone
+description: "Ingesting tables from a Dynamo DB instance in an Hakcney AWS account that is attached to the hub into the Data Platform landing zone"
```

Comment:
`an Hakcney` should be `a Hackney`
Also I'm a little confused by the word attached to the end of the sentence. Is it attached to both the hub and the Data Platform, or does it attach from the hub into the data platform (or vice versa)?

Reply:
I've removed this line, as James pointed out it doesn't actually matter if the account is on the hub.